src/main/java/sawkiewicz/task1/ShoppingCart.java

```diff
@@ -0,0 +1,30 @@
+package sawkiewicz.task1;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ShoppingCart {
+    private String id;
+    private List <CartItem> cartItems = new ArrayList<>();
+
+    public ShoppingCart() {
```

Comment:
this is default constructor, no need to write it

Reply:
removed